static/app/views/settings/organizationMembers/inviteRequestRow.spec.jsx

```diff
@@ -101,20 +104,20 @@ describe('InviteRequestRow', function () {
       />
     );
 
-    wrapper.find('button[aria-label="Approve"]').simulate('click');
+    userEvent.click(screen.getByRole('button', {name: 'Approve'}));
 
-    const modal = await mountGlobalModal();
-    modal.find('button[aria-label="Confirm"]').simulate('click');
+    renderGlobalModal();
+    userEvent.click(screen.getByTestId('confirm-button'));
```

Comment:
shall we be looking for `screen.getByRole('button', {name: 'Confirm'})`

Reply:
See https://github.com/getsentry/sentry/pull/40236#discussion_r999322630

I think the reason I haven't been doing this is sometimes the confirm button has a duplicate name. Idk, this is just fast and fine for now to get us off of enzyme